docs/RHDH-CONFIG.md

```diff
@@ -0,0 +1,472 @@
+    ```
+4. Dynamic plugins should be updated with the [Developer Hub plugins list](../dynamic-plugins/dh-plugins.yaml) with a pod update triggered and you may remove the temp file at this point
+
+#### Step 7: Updating RHDH Deployment
+
+Now you will need to make sure that all of the ArgoCD and Tekton tied resources are setup with the developer hub deployment.
+
+##### Step 7.1: Patch Kubernetes Service Account Token
+
+Run the following to patch in the Kubernetes Service Account Token that is needed for use under the Kubernetes dynamic plugin:
+
+```sh
+K8S_SA_SECRET_NAME=$(kubectl get secrets -n "$NAMESPACE" -o name | grep rhdh-kubernetes-plugin-token- | cut -d/ -f2 | head -1)
+K8S_SA_TOKEN=$(kubectl -n $NAMESPACE get secret $K8S_SA_SECRET_NAME -o yaml | yq '.data.token' -M -I=0)
+kubectl -n $NAMESPACE patch secret ai-rh-developer-hub-env \
+    --type 'merge' \
+    -p="{\"data\": {\"K8S_SA_TOKEN\": \"${K8S_SA_TOKEN}\"}}"
+```
+
+Notice that `K8S_SA_TOKEN` does not need encoding as the other literal sets, this is because when the value is fetched from the service account token secret it comes back already encoded.
+
+##### Step 7.2: ArgoCD Config and Secret
+
+Run the following to attach the ArgoCD ConfigMap and Secret:
+
+```sh
+kubectl -n $NAMESPACE get backstage ai-rh-developer-hub -o yaml | \
+        yq '.spec.application.appConfig.configMaps += [{"name": "argocd-config"}] | 
+            .spec.application.appConfig.configMaps |= unique_by(.name) |
+            .spec.application.extraEnvs.secrets += [{"name": "rhdh-argocd-secret"}] | 
+            .spec.application.extraEnvs.secrets |= unique_by(.name)' -M -I=0 -o=json | \
+        kubectl apply -n $NAMESPACE -f -
+```
+
+See [GitOps/ArgoCD Configuration](GITOPS-CONFIG.md#ai-rhdh-installer-manual-configuration) for information on creating the ArgoCD ConfigMap and Secret.
+
+### Pre-Existing Instance: Script Configuration
+
+For bringing a pre-existing instance to use with `configure-dh.sh`, ensure you gather the following information:
+1. Namespace of RHDH & ArgoCD instance
+2. Deployment name of RHDH
+3. ConfigMap name for your RHDH plugins
+   1. Typically `dynamic-plugins`
+4. (If applicable) Secret name for your RHDH environment variables
+   1. Only if bringing your own, if left blank or unset the script will create a new one
+
+You are able to store these values in environment variables. Set the following the use environment variables:
+- `$EXISTING_NAMESPACE`
+  - Name of target RHDH namespace
+- `$EXISTING_DEPLOYMENT`
+  - Name of target RHDH deployment
+- `$RHDH_PLUGINS`
+  - Name of the dynamic plugins ConfigMap
+- `$EXISTING_EXTRA_ENV_SECRET`
+  - Name of the extra environment variables Secret
+
+If using a [RHDH Operator](https://github.com/redhat-developer/rhdh-operator) instance, you will need to ensure the following fields are set to the developer hub entrypoint url under any of the app configs tied to your developer hub instance: 
```

Comment:
Is this saying you have to have a preconfigured rhdh with an app config? You can't just have a pre existing default setup?